atlasdb-feign/src/test/java/com/palantir/atlasdb/http/PollingRefreshableTest.java

```diff
@@ -181,7 +236,15 @@ public void cannotSetNegativeRefreshInterval() {
                     return newValue;
                 },
                 REFRESH_INTERVAL,
+                Function.identity(),
                 scheduler);
         return pollingRefreshable.getRefreshable();
     }
+
+    private long doubleOrThrowOnOne(long input) {
```

Comment:
nit: `private static`, surely?

Also this is fine, but I think actually having the behaviour be more explicit e.g. via a mock might be more readable for people picking this up later